pkg/api/v1/conversion_generated.go

```diff
@@ -26,7 +26,7 @@ import (
 	resource "k8s.io/kubernetes/pkg/api/resource"
 	unversioned "k8s.io/kubernetes/pkg/api/unversioned"
 	apiv1 "k8s.io/kubernetes/pkg/api/v1"
-	batchv1 "k8s.io/kubernetes/pkg/apis/batch/v1"
+	v1beta1 "k8s.io/kubernetes/pkg/apis/extensions/v1beta1"
```

Comment:
I've just switched to go 1.4 for the generation... Haven't got the time to look at why this is happening with 1.6 atm.